packages/trail-core/lib/index.js

```diff
@@ -9,19 +9,19 @@ const {parseDate, convertToTrail} = require('./trail')
 
 class TrailsManager {
   constructor (logger, pool) {
-    this.config = require('config')
-
     this.logger = logger || pino()
 
     if (typeof pool === 'undefined') {
+      const config = require('config')
```

Comment:
I see the point of moving here, in case a pool is provided. So kudos on this.

But the original code assigned `this.config` in order to enable developers to access it outside the constructor, if needed. Can you make sure the config it's still assigned to the object?

Reply:
I'm happy to restore it, if you're happy with the tradeoff: it creates an inconsistency, because developers providing their own pool will not have `this.config` set on their TrailsManager instance.

Reply:
As discussed online: let's remove it for now, since it was undocumented and untested.